tests/acceptance/builds/cancel-test.js

```diff
@@ -30,6 +29,8 @@ module('Acceptance | builds/cancel', function (hooks) {
     await click('[data-test-repo-actions-cancel-button]');
 
     assert.dom('[data-test-flash-message-text]').hasText('Build has been successfully cancelled.', 'cancelled build notification should be displayed');
-    assert.equal($('head link[rel=icon]').attr('href'), getFaviconUri('yellow'), 'expected the favicon data URI to match the one for running');
+    // Ember-test-helpers find does not work here
+    const iconHref = window.document.querySelector('head link[rel=icon]').getAttribute('href');
+    assert.equal(iconHref, getFaviconUri('yellow'), 'expected the favicon data URI to match the one for running');
```

Comment:
I think you can use [qunit-dom](https://github.com/simplabs/qunit-dom/blob/master/API.md#hasattribute) here:
```js
assert.dom('head link[rel=icon]').hasAttribute('href', getFaviconUri('yellow'));
```

Reply:
ahh nice, I don't think I had tried qunit-dom here :) Though when I tried it just now, it couldn't find the element

![element-exist-error](https://user-images.githubusercontent.com/1923979/70275514-476e0e80-177c-11ea-99be-6868bc27b8cb.png)

It's kind of strange, I would think that qunit-dom should be able to see it?